static/js/modules/tables.js

```diff
@@ -283,11 +283,16 @@ function filterSuccessUpdates(changeCount) {
         $label = $('[data-filter="typeahead"] li').last();
         filterAction = 'Filter added';
       }
-      else if ($(updateChangedEl).parent().hasClass('range__input')) {
+      else if ($(updateChangedEl).closest('.range').hasClass('range--currency')) {
         $label = $(updateChangedEl).closest('.range');
 
         filterAction = 'Filter applied';
       }
+      else if ($(updateChangedEl).closest('.range').hasClass('range--date')) {
+        $label = $('.date-range-grid');
```

Comment:
Let's scope this to the `updateChangedEl` in case there's multiple ones on a page.